backend/doc/api.md

```diff
@@ -55,6 +56,31 @@ Set-Cookie:  session_key=22fd293b6d850faabc8e3f167bcfc804d8713deed03bc15e5029434
 }
 ```
 
+#### With CURL
+```
+curl -i -u admin@admin.admin:admin@admin.admin http://127.0.0.1:6543/api/auth/whoami
```

Comment:
The other examples have a `$` before them, wouldn't it be relevant here too?

Reply:
Done in 86ca80f